cmd/builder/internal/builder/main.go

```diff
@@ -87,6 +87,9 @@ func Compile(cfg Config) error {
 	cfg.Logger.Info("Compiling")
 
 	args := []string{"build", "-ldflags=-s -w", "-trimpath", "-o", cfg.Distribution.Name}
+	if cfg.Distribution.DebugSymbols {
```

Comment:
I would prefer to have the contents of ldflags to be a var that is used in the args list. This way, the list can be the same independent of the flag's value, instead of repeating the same list, just to change the ldflags arg.

Reply:
done